src/style.css

```
@@ -0,0 +1,120 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
```

```suggestion
* {
```
сюди можна тільки box-sizing задавати, марджини краще в body і окремо в елементів сидувати чи в ресеті